backend/src/main.py

```
@@ -81,12 +92,31 @@ def internal_error(error):
+background_loop = None
+
+def start_async_loop():
+    """
+    Starts an asyncio event loop in a background thread for monitoring and auto-trading.
+    """
+    global background_loop
+    background_loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(background_loop)
 
+    # Schedule background tasks
+    background_loop.create_task(mempool_monitor_service.start_monitoring())
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `background_loop.create_task`

(RUF006)